lib/active_admin/resource_controller/decorators.rb

```diff
@@ -77,9 +77,9 @@ def self.wrap!(parent, name)
         # Draper::CollectionDecorator was introduced in 1.0.0
         # Draper::Decorator#collection_decorator_class was introduced in 1.3.0
         def self.find_collection_decorator(decorator)
-          if Dependency.draper? '>= 1.3.0'
+          if Dependency.draper?('>= 1.3.0') && decorator <= Draper::Decorator
```

Comment:
I removed all this stuff because we don't really need it. Can you rebase and adapt the PR? I guess it's a matter of saying

```suggestion
          if Dependency.draper? && decorator <= Draper::Decorator
```

and restoring the final branch.

Reply:
There we go, by removing the if here the expected interface for the decorator class changed to also require `collection_decorator_class`. But I think is a good tradeoff to have the code here cleaner.

Reply:
I wasn't meaning to suggest changing the interface for PORO decorators. I just meant to adapt the code to the lack of versions checks for `draper, with:

```ruby
def self.find_collection_decorator(decorator)
  if Dependency.draper? && decorator <= Draper::Decorator
    decorator.collection_decorator_class
  else
    decorator
end
```

Do you think we should require PORO decorators to define `collection_decorator_class`?

By the way, this remainded me that we need docs for this feature at `docs/11-decorators.md`.

Reply:
I proposed #6351. With that PR I believe we don't need any changes here in this file, nor to add a requirement for PORO decorators to define `collection_decorator_class`.

Reply:
Cool, yeah on a second thought I think making PORO define the `collection_decorator_class` to just return `self` might not be too elegant. Do you want to merge that refactor first and I'll add docs on how to use PORO presenters?

Reply:
Yeah! :+1:

Reply:
@brunvez we just merged in the change so you are good to rebase. Thanks!